lib/libutee/include/pta_device.h

```diff
@@ -0,0 +1,41 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) 2019, Linaro Limited
+ */
+
+#ifndef __PTA_DEVICE_H
```

Comment:
I think this deserves a comment to say what the pTA actually does (enumerate the pseudo-TAs that have the `TA_FLAG_KERNEL_DEVICE` flag).